angularjs/angular.d.ts

```diff
@@ -1024,7 +1024,7 @@ declare module ng {
             // Not garanteed to have, since it's a non-mandatory option
             //capacity: number;
         };
-        put(key: string, value?: any): void;
+        put<T>(key: string, value?: T): T;
```

Comment:
How did you go about getting those results?  When I run those examples I get the values back as expected:
http://plnkr.co/edit/VI1d3uyyRIPTI0rooNVO?p=preview